app/src/main/java/com/android/voyageur/ui/search/SearchScreen.kt

```diff
@@ -446,24 +446,26 @@ fun UserSearchResultItem(
             }
 
         Button(
-            onClick = { userViewModel.addContact(user.id) },
-            enabled = !isContactAdded,
-            shape = RoundedCornerShape(20.dp),
+            onClick = {
+              if (isContactAdded) {
+                userViewModel.removeContact(user.id) // Remove the contact if already added
+              } else {
+                userViewModel.addContact(user.id) // Add the contact if not added
+              }
+            },
             colors =
```

Comment:
The add contact button is no longer working, neither in the search view nor when trying to add from the profile view.

Reply:
I see, thanks for reminding me. Then @BogdanCDev could you change the button to "Requested" and when the request is accepted it should display "Added"/"Remove". Also, a toast with something like "Your friend request was sent"(in case of success) would be a good idea, and analogously when removing a friend(or a confirmation box).

Reply:
Thank you for your valuable suggestion! I’ve incorporated the requested feature into this PR.